src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -677,7 +678,7 @@ public void WhenTheUserChoosesNotToDeleteTheCatalogueSolution()
         [Then(@"only the Catalogue Solution with the unit is deleted from the order")]
         public async Task ThenOnlyTheCatalogueSolutionWithTheUnitIsDeletedFromTheOrderAsync()
         {
-            var orderItem = Context.Get<OrderItem>(ContextKeys.DeletedOrderItem);
+            var orderItem = Context.Get<Domain.OrderItem>(ContextKeys.DeletedOrderItem);
```

Comment:
Not sure how `Domain.` managed to sneak into this PR

Reply:
done